addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -292,21 +291,27 @@ odoo.define('website_slides.quiz', function (require) {
                     quizKarmaGain: slideData.quizKarmaGain,
                     quizAttemptsCount: slideData.quizAttemptsCount,
                 };
-                defs.push(new Quiz(self, slideData, slideData.quizData).attachTo($(this)));
+                var quiz = new Quiz(self, slideData, slideData.quizData)
+                self.quizWidgets.push(quiz);
+                defs.push(quiz.attachTo($(this)));
```

Comment:
$(this) works ?

Would have thought this.$el but ok :thinking:

Reply:
No, here we are in a jquery loop. $(this) will return the current jquery elem

Reply:
My bad, didn't read the whole method.